### PR TITLE
Typing: Runtime

### PR DESCRIPTION
**Pull request by asi1024**

### CODE REVIEW

File: pytorch_pfn_extras/runtime/_runtime.py [hunk 1]
```diff
@@ -1,7 +1,20 @@
+from typing import (
+    Any, Dict, Generator, List, Optional, Sequence, Tuple, Union, TYPE_CHECKING
+)
+
 import torch
 
 _RUNTIME_TAG_NAME = '_ppe_runtime'
 
+DeviceLike = Union[str, torch.device]
+Batch = Union[torch.Tensor, List[torch.Tensor],
```
Comment (kmaehashi) [inline pytorch_pfn_extras/runtime/_runtime.py:10]: Could you change to Any as discussed?

File: pytorch_pfn_extras/runtime/_runtime.py [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+              Tuple[torch.Tensor, ...], Dict[str, torch.Tensor]]
+Loader = Sequence[Batch]
```
Comment (kmaehashi) [inline pytorch_pfn_extras/runtime/_runtime.py:12]: I think this should be `Iterable` as `torch.utils.data.DataLoader` is not a sequence.

**Review by kmaehashi — CHANGES_REQUESTED** [review]

File: pytorch_pfn_extras/runtime/_runtime.py (hunk 2 above)
Comment (emcastillo) [reply pytorch_pfn_extras/runtime/_runtime.py:12]: Agree

**Review by emcastillo — APPROVED** [review]

**Pull request merged**